### ISSUE TITLE
tab pages are created/load twice

### ISSUE THREAD

**Issue by MarkHerhold** [ionic-team/ionic-conference-app#445]
**I'm submitting a ...**
[ x ] bug report

**Current behavior:**
Schedule, Map tabs are created/loaded twice and trigger lifecycle hooks twice.

I added a `console.log()` statement to both the schedule and map page `constructor()` and `ionViewDidLoad()` and see that both are called when the page is loaded.

```
...
schedule.ts:49 schedule list init
schedule.ts:53 ionViewDidLoad
...
schedule.ts:49 schedule list init
schedule.ts:53 ionViewDidLoad
```

**Expected behavior:**
I would expect that these pages would be created/loaded only once. This is problematic when trying to load data from an async source because calls are duplicated.

**Steps to reproduce:**
Clone the current version of the repo at commit 14a441ca1bf2d68d6a6e8d12b5807fd8b587bdaa and add a log statement to the `constructor` and `ionViewDidLoad()` lifecycle hooks in `schedule.ts`.

This is partially related to #334

**Comment by dyadyaJora** [comment]
@MarkHerhold 
Can you tell me - how to fix it in local project? Is it possible?

**Comment by MarkHerhold** [comment]
@dyadyaJora I have not attempted to fix this and don't have a production app (hence no need to fix). Sorry about that.

**Comment by brandyscarney** [comment]
I am going to close this as I am not able to reproduce it in latest on `master`. If you are still running into issues please create a new issue on the [Ionic repository](https://github.com/ionic-team/ionic/issues/new/choose). Thank you!
